libsignal-service-actix/src/websocket.rs

```diff
@@ -69,6 +69,7 @@ impl From<AwcWebSocketError> for ServiceError {
 async fn process<S: Stream>(
     socket_stream: S,
     mut incoming_sink: Sender<WebSocketStreamItem>,
+    keep_alive: bool,
```

Comment:
I think I would've added this in the generic WebSocket, but I'm not against. This way, the events are not generated, and that's maybe even better.

Reply:
Indeed, that's another approach, I kinda went quick and dirty with this one.

Reply:
EDIT: the events _are_ generated in the implementation? which is why I did it here, see https://github.com/whisperfish/libsignal-service-rs/pull/182/files/32c11d40437badcfa5cc96d2788b54ddbaca9c33#diff-72f8f8faf866526c65b6bf1d14cc57b52f1c3b5e49f003245839bb4435a8de7eR92

Reply:
the other way would be to match on requests, and discard the keep-alive?

Reply:
Totally fine.